api/types/client.go

```diff
@@ -154,28 +154,19 @@ type ImageBuildResponse struct {
 
 // ImageCreateOptions holds information to create images.
 type ImageCreateOptions struct {
-	// Parent is the image to create this image from
-	Parent string
-	// Tag is the name to tag this image
-	Tag string
-	// RegistryAuth is the base64 encoded credentials for this server
-	RegistryAuth string
+	Parent       string // Parent is the image to create this image from
```

Comment:
AFAIK, GoDoc does not even care where the comment is positioned; the body of the struct type is just printed out as-is ([example from `net/http`](https://godoc.org/net/http#Response) that contains both positions).
The standard library uses both positions, dependending on the length of the comment.

Where positioning might matter is IDE support. As far as I can remember, though, the Go plugin for IntelliJ IDEA handles both positions correctly.